assets/general/ltc/info.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "Litecoin",
+  "website": "https://litecoin.org",
+  "description": "Litecoin is a peer-to-peer Internet currency that enables instant, near-zero cost payments to anyone in the world",
+  "explorer": "https://litecoinspace.org",
+  "explorerTx": "https://litecoinspace.org/tx/${ID}",
+  "explorerAddress": "https://litecoinspace.org/address/${ID}",
+  "regexAddress": "^[LM3][A-Za-z0-9]{26,33}$",
+  "symbol": "LTC",
+  "type": "coin",
+  "decimals": 8,
+  "minBalance": 0.00000001,
```

Better to use a higher value to cover transaction fees.

```suggestion
  "minBalance": 0.001,
```